CONTRIBUTING.md

```diff
@@ -8,39 +8,36 @@ All contributions, bug reports, bug fixes, documentation improvements,
 enhancements and ideas are welcome.
 
 If you are simply looking to start working with the *pandas* codebase,
-navigate to the [GitHub "issues"
-tab](https://github.com/pydata/pandas/issues) and start looking through
-interesting issues. There are a number of issues listed under
-[Docs](https://github.com/pydata/pandas/issues?labels=Docs&sort=updated&state=open)
-and [Difficulty
-Novice](https://github.com/pydata/pandas/issues?q=is%3Aopen+is%3Aissue+label%3A%22Difficulty+Novice%22)
+navigate to the [GitHub "issues" 
+tab](https://github.com/pydata/pandas/issues) and start looking through 
+interesting issues. There are a number of issues listed under 
+[Docs](https://github.com/pydata/pandas/issues?labels=Docs&sort=updated&state=open) and [Difficulty 
+Novice](https://github.com/pydata/pandas/issues?q=is%3Aopen+is%3Aissue+label%3A%22Difficulty+Novice%22) 
 where you could start out.
 
 Or maybe through using *pandas* you have an idea of you own or are
 looking for something in the documentation and thinking 'this can be
 improved'...you can do something about it!
 
-Feel free to ask questions on [mailing
-list](https://groups.google.com/forum/?fromgroups#!forum/pydata)
+Feel free to ask questions on the [mailing list](https://groups.google.com/forum/?fromgroups#!forum/pydata)
```

Comment:
can you add a refernce to ``gitter`` as well, link is on the [readme](https://github.com/pydata/pandas/blob/master/README.md)